src/test/rpc/AccountObjects_test.cpp

```diff
@@ -575,8 +575,8 @@ class AccountObjects_test : public beast::unit_test::suite
         Account const gw{"gateway"};
         auto const USD = gw["USD"];
 
-        auto const features =
-            supported_amendments() | FeatureBitset{featureXChainBridge};
+        auto const features = supported_amendments() |
```

Comment:
Why is `featureXChainBridge` in `FeatureBitset` but not `featurePermissionedDomains`? 

IMO it should at least be consistent.

Reply:
fixed